src/Resource.ts

```diff
@@ -86,9 +86,10 @@ export class Resource<T extends SimpleObject> {
     body?: T;
     query?: SimpleObject;
   }): Promise<any> {
+    const url = path === null ? `${this.name}` : `${this.name}/${path}`;
```

Comment:
Change this because we were adding `/` to the end of the url even if there is no extra path string, some services/frameworks may not cope correctly with this.